core/client/src/test/java/alluxio/client/file/options/CreateFileOptionsTest.java

```diff
@@ -85,6 +89,7 @@ public void toThrift() {
         .isPersisted());
     Assert.assertEquals(mDefaultBlockSizeBytes, thriftOptions.getBlockSizeBytes());
     Assert.assertEquals(Constants.NO_TTL, thriftOptions.getTtl());
+    Assert.assertEquals(alluxio.thrift.TtlExpiryAction.Delete, thriftOptions.getTtlExpiryAction());
```

Comment:
would it make sense to prefix the thrift type with T to avoid the class name conflict?

(using `TTtlExpiryAction` for the thrift type and `TtlExpiryAction` for the Alluxio type)